api/app/signals/apps/signals/models/category.py

```diff
@@ -82,6 +82,10 @@ class Category(models.Model):
 
     name = models.CharField(max_length=255)
 
+    # SIG-4403 A category can be publicly accessible
+    public_name = models.CharField(max_length=255, null=True, blank=True)
```

Comment:
Should we allow this to be empty, or default it to a copy of `name`? (Adding some logic and a data migration.)

Reply:
It should be empty and not default to anything. This is something that we can handle in the FE or in the serializers. This field is also logged in the history.